lua/executor/executor.lua

```diff
@@ -385,6 +385,11 @@ M._show_split = function()
 end
 
 M.toggle_detail = function()
+  -- Don't attempt to show detail if no tests have been run.
+  if M._state.last_stdout == nil then
+    Output.write_data(nil, nil, nil, nil)
```

Comment:
is there a reason to call this rather than just print "No test run to show" here?

Reply:
Apologies for the late response!  I was trying to match the project's flow of printing through `Output.write_data()`.  If it's okay to break that flow, printing here directly works well.